src/runtime-rs/rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "1.57.0"
```

Comment:
It's better add a new line.
![image](https://user-images.githubusercontent.com/1212008/145750008-4b5c5969-55a1-455a-a311-863354361b9f.png)

Reply:
And about the Rust version, now we are using `versions.yaml` to manage it, if you this file, we will face keeping two places to manage. cc @jodh-intel @fidencio @bergwolf @jiangliu

Reply:
Add new line in first revision. I suggest keep rust-toolchian.toml.  Version defined in `versions.yaml` is only illustrative but in `rust-toolchian.toml` is compulsory. It would help maintain the consistency of the compilation environment